src/hermes/commands/process/base.py

```diff
@@ -17,6 +21,11 @@ class HermesProcessCommand(HermesCommand):
     """ Process the collected metadata into a common dataset. """
 
     command_name = "process"
+    settings_class = None
 
     def __call__(self, args: argparse.Namespace) -> None:
-        pass
+        self.args = args
+        # TODO: get harvested data
+        # TODO: Merge Datasets
```

Comment:
Also:
- Call pre-processors plugins (well, not really clear)
- Call processor plugins

These interfaces are not at all well-defined by now. We had different entry points, but I don't see this suitable for the new CLI without introducing new commands.

I cloud image that **processor** plugins have a pre-process stage (that is called before the merge) and a process stage (that is called after the merge). As we do not have lots to process for now, it's also fine to work without plugins here for now.